src/components/equations/ItemEffects.ts

```diff
@@ -18,4 +18,41 @@ export class ItemEffects<TGameStartr extends FullScreenPokemon> extends GeneralC
 
         move.remaining = Math.min(move.remaining + amount, move.uses);
     }
+
+    /**
+     * Increases a stat by 1.
+     *
+     * @param statistic   The statistic that is being modified.
+     */
+    private addBattleStat(statistic: IStatistic) {
+        statistic.current = Math.min(statistic.normal * 4, (statistic.normal / 2) + statistic.current);
+        console.log(statistic.current);
+    }
+
+    /**
+     * Opens dialog menu for using a battle item like X Attack / X Defend.
+     *
+     * @param pokemon   The pokemon who the X Item is being used on.
+     * @param statistic   The stat that is being modified by the item.
+     * @param type   The string representing the stat name.
+     */
+    public useXItem(pokemon: IPokemon, statistic: IStatistic, type: string) {
+
+        //TODO: Implement X Accuracy
+        this.gameStarter.menuGrapher.createMenu("GeneralText", {
+            deleteOnFinish: true,
+        });
+        this.gameStarter.menuGrapher.addMenuDialog(
+            "GeneralText",
+            [
+                [
+                    "%%%%%%%PLAYER%%%%%%% used X " + type.toUpperCase() + "!",
+                ],
+                pokemon.title.join("") + "'s " + type.toUpperCase() + " rose!",
```

Comment:
Nit: prefer a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). Preferring them for complicated string concatenation makes it less likely you'll eventually add two numbers within one by accident (I've done that way too many times).

Reply:
Talked about this. It's fine for now.